ACE/include/makeinclude/platform_android.GNU

```diff
@@ -166,15 +168,22 @@ ifeq (arm,$(findstring arm,$(ANDROID_ARCH)))
   # Set ABI specific flags
   ifeq ($(ANDROID_ABI),armeabi-v7a)
     FLAGS_C_CC += -march=armv7-a -mfloat-abi=softfp -mfpu=vfpv3-d16
-    LDFLAGS    += -march=armv7-a
+    LDFLAGS    += -march=armv7-a -Wl,--fix-cortex-a8
   endif
+
+  ifeq ($(ANDROID_ABI),arm64-v8a)
+    CROSS_COMPILE=aarch64-linux-android-
+    FLAGS_C_CC += -march=armv8-a #-mfloat-abi=softfp -mfpu=vfpv3-d16
+    LDFLAGS    += -march=armv8-a 
+  endif 
+
   ifeq ($(ANDROID_ABI),neon)
     FLAGS_C_CC += -march=armv7-a -mfloat-abi=softfp -mfpu=neon
-    LDFLAGS    += -march=armv7-a
+    LDFLAGS    += -march=armv7-a -Wl,--fix-cortex-a8
   endif
 
   # NDK says required for working around a CPU bug in some Cortex-A8 implementations
-  LDFLAGS += -Wl,--fix-cortex-a8
+  #LDFLAGS += -Wl,--fix-cortex-a8
```

Comment:
Move the comment up and refer to --fix-cortex-8, then you can remove this part here